rosbag2_compression/CMakeLists.txt

```diff
@@ -140,12 +140,14 @@ if(BUILD_TESTING)
   ament_add_gmock(test_sequential_compression_reader
     test/rosbag2_compression/test_sequential_compression_reader.cpp)
   target_include_directories(test_sequential_compression_reader PUBLIC include)
+  target_include_directories(test_sequential_compression_reader PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/../rosbag2_cpp/test")
```

Comment:
```suggestion
  target_include_directories(test_sequential_compression_reader PRIVATE "${CMAKE_SOURCE_DIR}/test")
```

Reply:
Unfortunately that won't work since its in `rosbag2_cpp` and not `rosbag2_compression`